resource/plugins/unreal/python/publisher/post_finalizers/unreal_assetinfo_publisher_post_finalizer.py

```diff
@@ -0,0 +1,113 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2023 ftrack
+import json
+import os
+
+import ftrack_api
+
+from ftrack_connect_pipeline import constants as core_constants
+from ftrack_connect_pipeline_unreal import plugin
+
+from ftrack_connect_pipeline_unreal.utils import (
+    custom_commands as unreal_utils,
+)
+from ftrack_connect_pipeline_unreal.constants import asset as asset_const
+
+
+class UnrealAssetInfoPublisherFinalizerPlugin(
+    plugin.UnrealPublisherPostFinalizerPlugin
+):
+    '''Plugin for finalizing the Unreal asset info publish process'''
+
+    plugin_name = 'unreal_assetinfo_publisher_post_finalizer'
+
+    def run(self, context_data=None, data=None, options=None):
+        '''Publish the non snapshot asset info to ftrack if exists'''
+
+        # Extract version ID from the run data
+        asset_version_id = asset_filesystem_path = param_dict = None
+        for comp in data:
+            if comp['type'] == core_constants.COMPONENT:
+                for result in comp['result']:
+                    if result['name'] == core_constants.EXPORTER:
+                        plugin_result = result['result'][0]
+                        asset_filesystem_path = plugin_result['result'][0]
+                        break
+            elif comp['type'] == core_constants.FINALIZER:
+                for result in comp['result']:
+                    if result['name'] == core_constants.FINALIZER:
+                        plugin_result = result['result'][0]
+                        asset_version_id = plugin_result['result'][
+                            'asset_version_id'
+                        ]
+                        break
+
+        if 'param_dict' in options:
+            # Given pre fetched by batch publisher
+            param_dict = options['param_dict']
+        else:
+            # Get asset info, must be run in main thread
+            if not asset_filesystem_path:
+                return {'message': 'No asset could be extracted from publish!'}
+
+            # Convert to game path
+            asset_path = unreal_utils.filesystem_asset_path_to_asset_path(
+                asset_filesystem_path
+            )
+
+            unused_dcc_object_name, param_dict = unreal_utils.get_asset_info(
```

Comment:
unused_dcc_object_name ?

Reply:
Standard I use is to prefix unused variables with "unused_" to highlight their role